source/develop/release-management/features/virt/headless-vm.html.md

```diff
@@ -13,32 +13,32 @@ feature_status: Merged
 
 ### Summary
 
-Support Headless VMs in Ovirt by adding an option to run a VM without any graphical console, for cases in which a graphical console is not needed for accessing this VM. This Headless VM will run without graphical and Video devices.
+Support Headless VMs in oVirt by adding an option to run a VM without any graphical console, for cases in which a graphical console is not needed for accessing this VM. This Headless VM will run without graphical and Video devices.
```

Comment:
Does it work on all cluster levels?  Yes

Does it get exported? Yes

What happens when you import a headless VM to a previous version? Not checked but I assume that the target imported VM will be set to default values for both video and graphics. Should be verified by the QE.

What about audio devices?
No special handling since the audio device should be disabled according to the "Soundcard enabled" configuration in console tab, which should be disabled for headless VM.